projects/Amlogic/devices/S912/packages/libhybris/package.mk

```diff
@@ -0,0 +1,39 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="libhybris"
+PKG_VERSION="54dd474"
+PKG_SHA256="cf4e3a38d9c9430987e67b06fe7a8c1d80cb36ebb979b52d73fb18522c02cb9a"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE="https://github.com/libhybris/libhybris"
+PKG_URL="https://github.com/libhybris/libhybris/archive/$PKG_VERSION.tar.gz"
+PKG_SOURCE_DIR="$PKG_NAME-$PKG_VERSION*/hybris"
+PKG_DEPENDS_TARGET="toolchain android-headers"
+PKG_SECTION="devel"
+PKG_SHORTDESC="libhybris: Allows to run bionic-based HW adaptations in glibc systems - libs"
+PKG_TOOLCHAIN="autotools"
+
+PKG_CONFIGURE_OPTS_TARGET="--enable-arch=$TARGET_ARCH \
+                           --with-default-egl-platform=fbdev \
+                           --with-android-headers=$BUILD/android-headers-25 \
+                           --with-default-hybris-ld-library-path=/system/lib \
+                           --enable-mali-quirks"
+post_makeinstall_target(){
```

Comment:
Please drop `post_makeinstall_target()` in favour of updated fbdev patch that you can find here:
https://raw.githubusercontent.com/kszaq/LibreELEC.tv/8.2.2.3/packages/devel/libhybris/patches/libhybris-0003-add-fbdev-window-h.patch

Reply:
I thought the idea going forward within LE was to reduce the number of patches?

Reply:
The patch I linked is already included in your PR but in a different and overblown version. I suggest replacing it with a better version that would also allow you to drop `post_makeinstall_target`.

Reply:
I've just noticed that thanks :) just waiting for compile to finish before updating PR.